masterlist.yaml

```diff
@@ -2837,6 +2837,17 @@ plugins:
         util: '[FO4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/fallout4/mods/2737)'
         itm: 429
 
+  - name: 'MarksmansMistress.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/56090/' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0xD05A81AD
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 19
+    clean:
+      - crc: 0xB47926A1
+        util: 'FO4Edit v4.0.4'
```

Comment:
We exclude cleaning data for dirty plugins after they're cleaned due to volatility in the checksum.

```suggestion
```